crowbar_framework/app/models/neutron_service.rb

```diff
@@ -15,6 +15,8 @@
 # limitations under the License.
 #
 
+require_relative '../helpers/barclamp/neutron_helper.rb'
```

Comment:
Helpers don't have to be mixed into models. They've got totally different responsibilities :(